interface/billing/billing_report.php

```diff
@@ -368,236 +387,293 @@ function MarkAsCleared(Type)
     <?php } ?>
-<input type="submit" class="subbtn" style="width:120px;" name="bn_hcfa_txt_file" value="<?php echo xla('CMS 1500 TEXT')?>"
- title="<?php echo xla('Making batch text files for uploading to Clearing House and will mark as billed')?>"
- onclick="MarkAsCleared(3)">
-<input type="submit" data-open-popup="true" class="subbtn" name="bn_mark" value="<?php echo xla('Mark as Cleared')?>" title="<?php echo xla('Post to accounting and mark as billed')?>">
-<input type="submit" data-open-popup="true" class="subbtn" name="bn_reopen" value="<?php echo xla('Re-Open')?>" title="<?php echo xla('Mark as not billed')?>">
-<!--
+<input type="submit" class="subbtn" name="bn_hcfa_txt_file"
+                value="&lt;?php echo xla('CMS 1500 TEXT')?&gt;"
+                title="&lt;?php echo xla('Making batch text files for uploading to Clearing House and will mark as billed')?&gt;"
+                onclick="MarkAsCleared(3)"> <input type="submit"
+                data-open-popup="true" class="subbtn-warning" name="bn_mark"
+                value="&lt;?php echo xla('Mark as Cleared')?&gt;"
+                title="&lt;?php echo xla('Post to accounting and mark as billed')?&gt;">
+                <input type="submit" data-open-popup="true" class="subbtn-warning"
+                name="bn_reopen" value="&lt;?php echo xla('Re-Open')?&gt;"
+                title="&lt;?php echo xla('Mark as not billed')?&gt;"> <!--
 <input type="submit" class="subbtn" name="bn_electronic_file" value="Make Electronic Batch &amp; Clear" title="<?php echo xla('Download billing file, post to accounting and mark as billed')?>">
 -->
 &nbsp;&nbsp;&nbsp;
 <?php echo xlt('CMS 1500 Margins'); ?>:
 &nbsp;<?php echo xlt('Left'); ?>:
 <input type='text' size='2' name='left_margin'
- value='<?php echo attr($left_margin); ?>'
- title='<?php echo xla('HCFA left margin in points'); ?>' />
+                value='&lt;?php echo attr($left_margin); ?&gt;'
+                title="&lt;?php echo xla('HCFA left margin in points'); ?&gt;" />
 &nbsp;<?php echo xlt('Top'); ?>:
 <input type='text' size='2' name='top_margin'
- value='<?php echo attr($top_margin); ?>'
- title='<?php echo xla('HCFA top margin in points'); ?>' />
-</span>
+                value='&lt;?php echo attr($top_margin); ?&gt;'
+                title="&lt;?php echo xla('HCFA top margin in points'); ?&gt;" />
+        </span>
 <?php } ?>
 
-</center>
-<input type='hidden' name='HiddenMarkAsCleared'  id='HiddenMarkAsCleared' value="" />
-<input type='hidden' name='mode' value="bill" />
-<input type='hidden' name='authorized' value="<?php echo attr($my_authorized); ?>" />
-<input type='hidden' name='unbilled' value="<?php echo attr($unbilled); ?>" />
-<input type='hidden' name='code_type' value="%" />
-<input type='hidden' name='to_date' value="<?php echo attr($to_date); ?>" />
-<input type='hidden' name='from_date' value="<?php echo attr($from_date); ?>" />
+</span> <input type='hidden' name='HiddenMarkAsCleared'
```

Comment:
Now that you point it out, it also has done some bizaar things with the php tags